.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```

Comment:
What do you think about adding a section like:

## Confirmation

- [ ] I have read all the above instructions.
- [ ] I have fulfilled the above and created my PR accordingly.
- [ ] My code is upto standards. 
- [ ] This PR only includes specific changes from the relevant issue it will be fixing.

Reply:
I don't think it would be a good idea. It would put extra pressure on contributors